sources/models/src/lib.rs

```diff
@@ -237,3 +237,13 @@ struct Metadata {
     md: SingleLineString,
     val: toml::Value,
 }
+
+///// Bootstrap Containers
+
+#[model]
+struct BootstrapContainer {
+    source: Url,
+    mode: ValidContainerMode,
```

Comment:
Let's scope this type to bootstrap containers:
```suggestion
    mode: ValidBootstrapContainerMode,
```

Reply:
Done